docs/reference/func.md

```diff
@@ -4,7 +4,7 @@ Serverless functions
 
 ### Synopsis
 
-Serverless functions v0.0.0-source-2022-09-20T08:13:19&#43;02:00
+Serverless functions v0.0.0-source-2022-09-21T09:06:26&#43;01:00
```

Comment:
After rebasing the timestamp should not be included here.

Reply:
done